contracts/StablecoinConverter.sol

```diff
@@ -70,6 +102,15 @@ contract StablecoinConverter is EpochTokenLocker {
         numTokens++;
     }
 
+    /** @dev a user facing function used to place limit sell orders in auction with expiry defined by batchId
+      * @param buyToken id of token to be bought
+      * @param sellToken id of token to be sold
+      * @param isSellOrder - TODO - remove with
```

Comment:
Would it make sense to rebase on top of the PR that already removed this?

Reply:
I plan to merge master into this branch before continuing... yes.